src/System.Threading.Thread/tests/System.Threading.Thread.Tests.csproj

```diff
@@ -27,6 +27,7 @@
     <ProjectReference Include="..\pkg\System.Threading.Thread.pkgproj">
       <OSGroup>$(InputOSGroup)</OSGroup>
     </ProjectReference>
+    <ProjectReference Include="..\..\System.Threading\tests\System.Threading.Tests.csproj" />
```

Comment:
Should we just move ThreadTestHelpers.cs to Common\tests\System\Threading, and use it in both test libs?